main/main.c

```diff
@@ -2177,6 +2177,7 @@ int php_module_startup(sapi_module_struct *sf, zend_module_entry *additional_mod
 	REGISTER_MAIN_LONG_CONSTANT("PHP_INT_MAX", ZEND_LONG_MAX, CONST_PERSISTENT | CONST_CS);
 	REGISTER_MAIN_LONG_CONSTANT("PHP_INT_MIN", ZEND_LONG_MIN, CONST_PERSISTENT | CONST_CS);
 	REGISTER_MAIN_LONG_CONSTANT("PHP_INT_SIZE", SIZEOF_ZEND_LONG, CONST_PERSISTENT | CONST_CS);
+	REGISTER_MAIN_LONG_CONSTANT("PHP_FD_SETSIZE", FD_SETSIZE, CONST_PERSISTENT | CONST_CS);
```

Comment:
I agree that the name is somewhat cryptic, but it just exposes the low-level [FD_SETSIZE](http://www.delorie.com/gnu/docs/glibc/libc_248.html) to userland. Note that some functions (e.g. stream_select()) [raise the following warning](https://github.com/php/php-src/blob/php-7.0.8/main/network.c#L1169-L1191) when this limit is exceeded:

> You MUST recompile PHP with a larger value of FD_SETSIZE.
> […]

So in my opinion the name is sensible. Maybe the PHP_ prefix should be removed.